package-lock.json

```diff
@@ -261,8 +261,7 @@
     "abbrev": {
```

Comment:
I'm confused about why so many unrelated packages here have had their properties changed. @joelanman it might be best if you discard this package-lock and install `scss-to-json` from npm again. Give me a shout if you have any issues.